crates/orderbook/src/database/fees.rs

```diff
@@ -139,17 +139,6 @@ mod tests {
     use model::order::OrderKind;
     use primitive_types::H160;
 
-    // Convenience to allow using u32 in tests instead of the struct
-    impl From<u32> for UnsubsidizedFee {
```

Comment:
Moved to the definition of UnsubsidizedFee because I found it confusing that this was in an unrelated file.